snippets/logging-clusteradmin-access-logs-snip.adoc

```diff
@@ -0,0 +1,37 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+// * modules/cluster-logging-loki-deploy.adoc
+//
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+The error `Parse error: input size too long (XXXX > 5120)` comes when a `cluster-admin` user tries to query application logs for multiple namespaces, where the sum of total characters of all the namespaces in the cluster is more than 5120. To achieve better control over access to logs in LokiStack, the `cluster-admin` user should be a part of the `system:cluster-admins` group. If the group does not exist, create a new group and add the `cluster-admin` users.
```

Comment:
I'm confused about whether we're talking about a user literally named `cluster-admin` or if we're talking about creating a `system:cluster-admin` group that contains all of the users with admin permissons. The text here also references a group named `system:cluster-admins` which is not created or described in the procedure that follows.

Reply:
@kowen-rh Users with `cluster-admin` role will face the above mentioned error when they try to access logs in LokiStack. They should be added to a `cluster-admin` group to avoid so. Reference: [LOG-4236
](https://issues.redhat.com/browse/LOG-4236) 
I have made some changes based on your comment. PTAL and let me know if the disclaimer matches with the procedure now. Thanks! :)